e2e/testdata/fn-render/invalid-configmap-fnconfig/.expected/config.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+exitCode: 1
```

Comment:
how the CLI fails is important, and need to be tested as part of https://github.com/GoogleContainerTools/kpt/issues/1666

Reply:
Yes. I plan to tackle that in the PR where I am improving the error handling.